core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/api/generateCode.kt

```diff
@@ -19,6 +94,16 @@ public inline fun <reified T> DataFrame<T>.generateCode(
         extensionProperties = extensionProperties,
     )
 
+/**
+ * @include [CommonGenerateCodeDocs]
+ * {@set [CommonGenerateCodeDocs.RECEIVER] [DataFrame's][this] [schema][DataFrameSchema]}
+ * {@set [CommonGenerateCodeDocs.TYPES] interfaces and extension properties}
+ *
+ * @include [Params.MarkerName]
+ * @include [Params.Fields] Default is `true`.
+ * @include [Params.ExtensionProperties] Default is `true`.
+ * @include [Params.Visibility] Default is [MarkerVisibility.IMPLICIT_PUBLIC].
+ */
 public fun <T> DataFrame<T>.generateCode(
```

Comment:
For `generateCode`, add more info about extension properties.

Reply:
I put it in the parameter description